engine/api/src/main/scala/pl/touk/nussknacker/engine/api/context/ValidationContext.scala

```diff
@@ -53,4 +56,38 @@ case class ValidationContext(localVariables: Map[String, TypingResult] = Map.emp
       case Some(ctx) => Valid(ctx)
       case None => Invalid(NoParentContext(nodeId.id)).toValidatedNel
     }
+}
+
+/**
+  * Right now we have a few different ways to name output param in node.. OutputVar allows us to skip using strings.
+  * @TODO: Provide one way for naming output in all nodes.
+  */
+case class OutputVar(fieldName: String, outputName: String)
+
+object OutputVar {
+
+  val DefaultFieldName = "varName"
+
+  val EnricherFieldName = "output"
+
+  val SubprocessFieldName = "outputName"
+
+  val SwitchFieldName = "exprVal"
+
+  val CustomNodeFieldName = "outputVar"
+
+  def apply(outputName: String): OutputVar =
```

Comment:
considering the above, maybe let's change method name from apply to variable?

Reply:
Yep, I've changed it.